IdentifiabilityODE.mpl

```diff
@@ -351,12 +351,21 @@ IdentifiabilityODE := proc(system_ODEs, params_to_assess, {p := 0.99, infolevel
     printf("%s %a\n", `Not identifiable parameters:                      `, map(x -> ParamToOuter(x, all_vars), select(p -> not p in theta_l, theta))):
     printf("===============\n\n"):
   end if:
-
-  table([
+  output := table([
     globally = {op(map(x -> ParamToOuter(x, all_vars), theta_g))},
     locally_not_globally = {op(map(x -> ParamToOuter(x, all_vars), select(p -> not p in theta_g, theta_l)))},
     non_identifiable = {op(map(x -> ParamToOuter(x, all_vars), select(p -> not p in theta_l, theta)))}
   ]):
+  for var in map(ParamToInner, output[globally]) do
+ 	  printf("%s %a %s %a\n",`The number of solutions for`, var, `is`, 1):
+  end do:

```

Comment:
I suggest to print anything only for high enough infolevel

Reply:
Added infolevel>1 check and the numbers of solutions are stored in a hash-table